src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -6265,51 +6263,17 @@ class C
             int exitCode = csc.Run(outWriter);
             Assert.Equal(0, exitCode);
 
-            var patched = Regex.Replace(outWriter.ToString().Trim(), "version \\d+\\.\\d+\\.\\d+(\\.\\d+)?", "version A.B.C.D");
-            patched = ReplaceCommitHash(patched);
+            var patched = Regex.Replace(outWriter.ToString().Trim(), "version \\d+\\.\\d+\\.\\d+(\\.\\d+)?-[\\w\\d.-]+", "version A.B.C.D-E");
             Assert.Equal(@"
-Microsoft (R) Visual C# Compiler version A.B.C.D (HASH)
+Microsoft (R) Visual C# Compiler version A.B.C.D-E
 Copyright (C) Microsoft Corporation. All rights reserved.".Trim(),
                 patched);
             // Privately queued builds have 3-part version numbers instead of 4.  Since we're throwing away the version number,
```

Comment:
Is this comment still applicable at all?

Reply:
I'm not sure. Does Arcade have a stable design for version numbers that we can depend on?